rasa/nlu/classifiers/embedding_intent_classifier.py

```diff
@@ -817,6 +817,16 @@ def _calculate_message_sim(
         # transform sim to python list for JSON serializing
         return label_ids, message_sim.tolist()
 
+    @staticmethod
+    def _text_features_present(session_data: SessionDataType) -> bool:
+        return np.array(
+            [
+                f.toarray().any() if isinstance(f, scipy.sparse.spmatrix) else f.any()
```

Comment:
it should be possible to check if sparse matrix is empty without converting it to numpy